### PR TITLE
introduce androidx support on the android docs

### PR DESCRIPTION
**Pull request by marandaneto**
Fix https://github.com/getsentry/sentry-android/issues/472

### CODE REVIEW

File: src/collections/_documentation/platforms/android/index.md [hunk 1]
```diff
@@ -654,6 +654,21 @@ And, add to `multidex-config.pro` the following lines:
 
 If you experience issues like `Could not find class` on devices running the `Dalvik VM`, you may expand the above rules to keep the necessary classes in the main dex file.
 
+### AndroidX Support
+
+We're using the AndroidX libraries for detecting when the Application is either on background or foreground, this is necessary for having an accurate result across all the Android OS versions.
```
Comment (HazAT) [inline src/collections/_documentation/platforms/android/index.md:659]:
```suggestion
We're using the AndroidX libraries for detecting when the Application is either in background or in foreground, this is necessary for having an accurate result across all the Android OS versions.
```

File: src/collections/_documentation/platforms/android/index.md [hunk 2]
```diff
@@ -654,6 +654,21 @@ And, add to `multidex-config.pro` the following lines:
...
+
+We check on runtime for its availability, so if you're not using the AndroidX libraries, you can remove them from Sentry's transitive dependencies.
```
Comment (HazAT) [inline src/collections/_documentation/platforms/android/index.md:661]:
```suggestion
We check during runtime for availability, so if you're not using the AndroidX libraries, you can remove them from Sentry's transitive dependencies.
```

File: src/collections/_documentation/platforms/android/index.md [hunk 3]
```diff
@@ -654,6 +654,21 @@ And, add to `multidex-config.pro` the following lines:
 
 If you experience issues like `Could not find class` on devices running the `Dalvik VM`, you may expand the above rules to keep the necessary classes in the main dex file.
 
+### AndroidX Support
+
+We're using the AndroidX libraries for detecting when the Application is either in background or in foreground, this is necessary for having an accurate result across all the Android OS versions.
```
Comment (bruno-garcia) [inline src/collections/_documentation/platforms/android/index.md:659]:
```suggestion
We're using the AndroidX libraries for detecting when the Application is either in the background or in the foreground, this is necessary for having an accurate result across all the Android OS versions.
```

File: src/collections/_documentation/platforms/android/index.md [hunk 4]
```diff
@@ -654,6 +654,21 @@ And, add to `multidex-config.pro` the following lines:
...
+
+We check during runtime for availability, so if you're not using the AndroidX libraries, you can remove them from Sentry's transitive dependencies.
```
Comment (bruno-garcia) [inline src/collections/_documentation/platforms/android/index.md:661]:
```suggestion
We check at runtime for availability, so if you're not using AndroidX libraries, you can remove them from Sentry's transitive dependencies.
```

File: src/collections/_documentation/platforms/android/index.md [hunk 5]
```diff
@@ -654,6 +654,21 @@ And, add to `multidex-config.pro` the following lines:
 
 If you experience issues like `Could not find class` on devices running the `Dalvik VM`, you may expand the above rules to keep the necessary classes in the main dex file.
 
+### AndroidX Support
+
+We're using the AndroidX libraries for detecting when the Application is either in the background or in the foreground, this is necessary for having an accurate result across all the Android OS versions.
```
Comment (MimiDumpling) [inline src/collections/_documentation/platforms/android/index.md:659]:
```suggestion
Sentry uses the AndroidX libraries for detecting when the Application is either in the background or in the foreground. This is necessary for having an accurate result across all the Android OS versions.
```

**Comment by marandaneto** [comment]
> A couple small edits. Thank you! 🌻

thanks @MimiDumpling 🌻 :)

**Pull request merged**